gradle/component.gradle

```diff
@@ -25,7 +25,7 @@ kotlin {
             binaries {
                 // Use this entry point to turn the thread tests are run to a background thread instead of the main thread
                 // This better allows testing the main dispatcher, and testing cross thread access
-                binaries.getTest("DEBUG").freeCompilerArgs += ["-e", "com.splendo.kaluga.test.mainBackground"]
+                binaries.getTest("DEBUG").freeCompilerArgs += ["-e", "com.splendo.kaluga.test.base.mainBackground"]
```

Comment:
Users will have to change this too 😬 I hope you just get an error if this is something wrong.

One for the release notes, cc: @cikpis